### DISCUSSION TITLE
GitHub Student Developer Pack Expiry

### DISCUSSION THREAD

**Discussion by kodikaop** [community/community#53624] [GitHub Education]
Hello, will your unused offers from GitHub Student Developer Pack (GSDP) expire when your GitHub Pro subscription free (with GSDP) expires?

**Comment by samus-aran** [comment]
Hi @kodikaop 

If you haven't redeemed offers once your GSDP valid verification expires then you will not be able to use them. If you have redeemed offers which exceeds the GSDP valid verification session then depends on the offer you will continue to use them till the end of that particular offer has been reached.

Hope this helps! 👍
